opentelemetry/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - [#1623](https://github.com/open-telemetry/opentelemetry-rust/pull/1623) Add global::meter_provider_shutdown
 - [#1640](https://github.com/open-telemetry/opentelemetry-rust/pull/1640) Add `PropagationError`
+- [#1701](https://github.com/open-telemetry/opentelemetry-rust/pull/1701) Stabilize `Gauge`
```

Comment:
```suggestion
- [#1701](https://github.com/open-telemetry/opentelemetry-rust/pull/1701) Stabilize `Gauge`
```

^ Will this give the wrong impression - entire OTel Rust is alpha only!

Suggestion:
`Gauge` no longer requires `otel-unstable` feature flag, as OpenTelemetry specification for Gauge instrumentation is stable.

Reply:
Updated.